packages/flutter_tools/lib/src/artifacts.dart

```diff
@@ -159,10 +159,10 @@ class CachedArtifacts extends Artifacts {
         // android_arm in profile mode because it is available on all supported host platforms.
         return _getAndroidArtifactPath(artifact, TargetPlatform.android_arm, BuildMode.profile);
       case Artifact.flutterTester:
-        if (platform == TargetPlatform.windows_x64)
-          throw new UnimplementedError('Artifact $artifact not available on platfrom $platform.');
-        continue fallThrough;
-      fallThrough:
+        final String engineArtifactsPath = cache.getArtifactDirectory('engine').path;
+        final String platformDirName = getNameForTargetPlatform(platform);
+        String path = fs.path.join(engineArtifactsPath, platformDirName, _artifactToFileName(artifact));
+        return (platform == TargetPlatform.windows_x64) ? (path + '.exe') : path;
```

Comment:
Are you sure this needs the `.exe`? For gen_snapshot for example we just provide the file name without ending and I believe the process lib figures out which ending is actually required...